.jshintrc

```diff
@@ -1,28 +1,68 @@
 {
-    "browser": true,
+    "bitwise": false,
```

Comment:
FYI, we do have bitwise operators in portions of the code.  Turn this off to find them...